app/javascript/packs/base.jsx

```diff
@@ -144,3 +144,25 @@ getInstantClick().then((spa) => {
 });
 
 initializeNav();
+
+async function loadCreatorSettings() {
+  try {
+    const [{ CreatorSettingsController }, { Application }] = await Promise.all([
+      import('@controllers/creator_settings_controller'),
+      import('@hotwired/stimulus'),
+    ]);
+
+    const application = Application.start();
+    application.register('creator-settings', CreatorSettingsController);
+  } catch (error) {
+    Honeybadger.notify(
+      `Error loading the creator settings controller: ${error.message}`,
+    );
+  } finally {
+    // TODO: Friendly error message if the controller fails to load
```

Comment:
@nickytonline will you be adding this to this PR?

Reply:
As discussed, I'm going to remove this. It's being a little too over defensive in terms of coding.